admin/spec/service/data-access/group.data.service.spec.js

```diff
@@ -19,11 +20,11 @@ describe('GroupDataService', () => {
 
   afterEach(() => sandbox.restore())
 
-  describe('#groupDataService', () => {
+  describe('#fetchGroups', () => {
     let mock
 
     beforeEach(() => {
-      mock = sandbox.mock(Group).expects('find').chain('sort').chain('lean').chain('exec').resolves(groupsMock)
+      mock = sandbox.mock(Group).expects('find').chain('collation').chain('sort').chain('lean').chain('exec').resolves(groupsMock)
```

Comment:
just a reminder to modify this if you remove the collation call from the data service 🎗